src/firebase.js

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import firebase from 'firebase/app';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'firebase/firestore';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'firebase/database';
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
+  apiKey: 'AIzaSyC90q2dSY4tlg2BXG-H1IVQCtdjMjVwXe4',
+  authDomain: 'sol-pomo.firebaseapp.com',
+  projectId: 'sol-pomo',
+  storageBucket: 'sol-pomo.appspot.com',
+  messagingSenderId: '439767660233',
+  appId: '1:439767660233:web:b32d2ca986b4beb1217f18',
```

Comment:
Check out Motaz's PR, I left him a comment about this. In essence, you should not store this in here, but instead use dotenv.

Reply:
I check Motaz's PR, but I couldn't find your comment about this.
I also ask Motaz about that and he didn't know. @mujz

Reply:
I am searching about ``how to use dotenv in node js``.